dependencies.list

```diff
@@ -1,4 +1,4 @@
 PACKAGE_NAME=realm-js
-VERSION=10.0.0-beta.10
-REALM_CORE_VERSION=10.0.0-beta.4
-REALM_SYNC_VERSION=10.0.0-beta.6
+VERSION=10.0.0-beta.11
```

Comment:
`10.0.0-beta.9` is the latest beta release on https://www.npmjs.com/package/realm & changelog marks updated as `10.0.0-beta.10`